app/src/main/java/org/bspb/smartbirds/pro/forms/convert/fields/SingleChoiceConverter.java

```diff
@@ -31,11 +31,6 @@ public void convert(Map<String, String> csv, JsonObject json, Set<String> usedCs
         } else {
             JsonObject label = new JsonObject();
             label.addProperty(locale, value);
```

Comment:
This allows submitting value from one language as another language - if user switched language between saving the csv and uploading.